src/music/music.repository.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@nestjs/common';
+import { CreateMusicDto } from './dto/create-music.dto';
+import { UpdateMusicDto } from './dto/update-music.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Music } from './entities/music.entity';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class MusicRepository {
+    constructor(@InjectRepository (Music)
+private MusicRepository: Repository <Music>) {}
+  async create(createMusicDto: CreateMusicDto) {
+    
+    const music =  await this.MusicRepository
+    .createQueryBuilder()
+    .insert()
+    .into(Music)
```

Comment:
When you describe the repository in the controller it is no longer necessary to use into

Reply:
Done